modules/is-restorable/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./dist/index.js",
   "jsnext:main": "./jsnext.js",
   "scripts": {
-    "test": "kocha --require babel-register test/*.js --color always"
+    "test": "../../node_modules/.bin/nyc --nycrc-path ../../.nycrc kocha --require babel-register test/*.js --color always"
```

Comment:
Each module should not depend on the root package. In this case, you should install nyc and .nycrc at every module.